lib/widgets/fields/padding_field.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_quest/widgets/fields/padding_types.dart';
+
+class PaddingField extends StatefulWidget {
+  final void Function(EdgeInsets) onChanged;
+
+  const PaddingField({
+    super.key,
+    required this.onChanged,
+  });
+
+  @override
+  State<PaddingField> createState() => _PaddingFieldState();
+}
+
+class _PaddingFieldState extends State<PaddingField> {
+  int selectedOption = 1;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            CustomRadioButton(
+              text: "All",
+              index: 1,
```

Comment:
@TayybaFunsol @hassanraza80775 Why are we still using indexes when even when discussed various benefits of enums over indexed in this [this article](https://medium.com/@s4.ali/flutter-code-review-dos-and-don-ts-and-best-practices-2-b856cf28b87a)?